src/openrct2/actions/LargeSceneryPlaceAction.hpp

```diff
@@ -313,7 +313,7 @@ DEFINE_GAME_ACTION(LargeSceneryPlaceAction, GAME_COMMAND_PLACE_LARGE_SCENERY, La
             TileElement* newTileElement = tile_element_insert(
                 { curTile.x / 32, curTile.y / 32, zLow }, quarterTile.GetBaseQuarterOccupied());
             Guard::Assert(newTileElement != nullptr);
-            map_animation_create(MAP_ANIMATION_TYPE_LARGE_SCENERY, curTile.x, curTile.y, zLow);
+            map_animation_create(MAP_ANIMATION_TYPE_LARGE_SCENERY, { curTile, zLow * 8 });
```

Comment:
I think there should be `z` and then `zLow = z/8`, there are multiple places above doing `zLow * 8`

(This will probably apply to other files as well, but not sure if worth doing in this PR)

Reply:
I don't think I'll address that in this PR. I need to come up with a standard for the z's. I guess we could standardise around using `z` for big Z and `baseZ`, `clearZ` or `lowZ`,`highZ` for small Z.

Reply:
Fair enough